infra/helper.py

```diff
@@ -894,6 +905,36 @@ def build_fuzzers(args):
       for sanitizer, child_dir in sanitized_binary_directories)
 
 
+def ood_build_fuzzers(args):
```

Comment:
Should we just call this build_fuzzbench_fuzzer to be more descriptive to users?